pkg/install/1-installresources.go

```diff
@@ -668,9 +651,20 @@ func (i *Installer) installResources(ctx context.Context) error {
 			return err
 		}
 	}
-
 	{
-		ipAddress := lbIP.String()
+		var ipAddress string
+		{
+			// Discover what private IP address was assigned to internal LB
+			lbClient := mgmtnetwork.NewLoadBalancersClient(i.env.SubscriptionID())
+			lbClient.Authorizer = i.fpAuthorizer
+			resourceGroupIDSplit := strings.Split(i.doc.OpenShiftCluster.Properties.ClusterProfile.ResourceGroupID, "/")
```

Comment:
search in the code `azure.ParseResourceID(oc.ID)`

Reply:
I am trying to extract the resource group name from the resource group ID. `ParseResourceID` can only parse resource IDs. I can write a new function similar to `ParseResourceID` called `ParseResourceGroupID`. What do you think about that idea, and where would you suggest I put it?